CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ option(ARMA_EXTRA_DEBUG "Compile with extra Armadillo debugging symbols." OFF)
 option(TEST_VERBOSE "Run test cases with verbose output." OFF)
 option(BUILD_TESTS "Build tests." ON)
 option(BUILD_CLI_EXECUTABLES "Build command-line executables." ON)
-option(DISABLE_DOWNLOADS "Disable downloads of dependencies during build." OFF)
+option(DISABLE_DOWNLOADS "Disable downloads of dependencies during build." ON)
```

Comment:
The use of a double negative here can be confusing -- using "ON" to actually disable something.  

For ease of understanding (and easier maintenance) it's generally best to have a simple on/off for a given functionality.  I recommend to change the option to `ENABLE_DOWNLOADS` or more accurately `DOWNLOAD_DEPENDENCIES`.  

Using "ON" would then clearly mean to enable downloading the dependencies.

Reply:
Agreed, this would be a nice improvement. :+1: